packages/gui/src/components/ModeSelector.tsx

```diff
@@ -39,7 +37,7 @@ const ModeSelector = () => {
   const [open, setOpen] = useState(false)
   const [disabled, setDisabled] = useState(false)
   const [showModeDialog, setShowModeDialog] = useState(false)
-  const { modeName, _setModeName, prevMode, modes } = useContext(ModeContext)
+  const { modeName, _setModeName, modes } = useContext(ModeContext)
```

Comment:
@christian-bromann I did remove it. You're referring this `prevMode` correct?

Reply:
Oh, I think you mean remove it from context itself. Yeah I can do that too.

Reply:
I don't think we need it at this point anymore.